contrib/utilities/check_parameter_files_in_examples.sh

```diff
@@ -45,8 +45,9 @@ lethe_fluid_particles=("examples/unresolved-cfd-dem")
 for folder in ${lethe_fluid_particles[@]}; do
   for file in $(find "$folder" -type f -name "*.prm"); do
     echo $file
-    # If generator is in the file name this is a lethe-particles file
-    if [[ "$file" == *"generator"* ]];then
+    # If "generator" or "particle" is in the filename this is a
+    # lethe-particles file
+    if [[ "$file" == *"generator"* ]] || [[ "$file" == *"particle"* ]];then
```

Comment:
I do have 2 DEM parameter files, naming it generator doesn't really make sense

Reply:
well it could be generator_part1 and generator_part2 because in the end they generate the inital condition. I just don't want to create a new nomenclature with two names. This was already painful to implement initially to have the correct parameter testing. Let's ensure there's only one token, not two.

Reply:
I changed it to *particles* and renamed the DEM .prm file as `*action*-particles.prm` (which they were all renamed `packing-particles.prm`. 
In my opinion "particles" is more intuitive tham "generator". I did not know about the "generator" convention and not using this term makes the parameter-check crashes without any tips on why. (I was getting a boundary parameter files error). I had to dig a bit to get that there was a script looking for "generator". I mean, the script and the test are really smart and useful I like it, and a big thanks! And thinking about how to implement a way to sort the DEM .prm is nice, but was not obvious to find as a "exemple creator" perspective. We can add a note in the contributing file, but even with that, this is a tinny info in lots of information, easy to miss it. 
Anyway, if you have any comments on that, just them me :)